docs/src/adoc/index.adoc

```diff
@@ -4069,6 +4064,7 @@ Be careful using this feature! This can become a vector for SQL injection
 attacks if you fail to
 link:https://xkcd.com/327/[sanitize your database inputs^].
 
+////
```

Comment:
Ditto for this section--the header by itself only teases but doesn't tell the reader anything about it.